packages/react-instantsearch-dom/src/widgets/Answers.js

```diff
@@ -64,6 +65,7 @@ export default function Answers({
     setIsLoading(true);
     runConcurrentSafePromise(
       searchIndex.findAnswers(_query, queryLanguages, {
+        ...extraParameters,
```

Comment:
@eunjae-lee  I wonder if this is not a bit confusing to have some parameters top level (`nbHits`, `query`, ect) and some nested in `extraParameter` (eg `params` and `threshold`). I wonder if it would make more sens to have them all at the same level ?

Reply:
@RaseMotte 
The yesterday's version I sent you was 

```jsx
<Answers extraParameters={{
  threshold: xxx,
  params: {
    ...
  }
}} />
```

I'm thinking to change it to

```jsx
<Answers threshold={xxx} params={{
  filters: ...
  , whatever search parameters, ...
}} />
```

I guess this is also what you're talking about. What do you think?